ldms/src/sampler/dstat/parse_stat.c

```diff
@@ -127,7 +127,7 @@ int parse_proc_pid_stat(struct proc_pid_stat *s, const char *pid) {
 	commstart++;
 	if ((commend - commstart) >= COMM_SZ)
 		return ENAMETOOLONG;
-	strncpy(s->comm, commstart , COMM_SZ);
+	snprintf(s->comm, COMM_SZ, "%s", commstart);
```

Comment:
Are these the warnings related to the size being on the source buffer instead of the destination buffer?

Reply:
I think at this spot it warned about not terminating with '\0'.